crates/libs/bindgen/src/handles.rs

```diff
@@ -20,26 +20,42 @@ pub fn gen_sys_handle(def: &TypeDef, gen: &Gen) -> TokenStream {
 pub fn gen_win_handle(def: &TypeDef, gen: &Gen) -> TokenStream {
     let name = def.name();
     let ident = gen_ident(def.name());
-    let signature = gen_signature(def, gen);
+    let underlying_type = underlying_type(def);
+    let signature = gen_default_type(&underlying_type, gen);
+    let check = if underlying_type.is_pointer() {
+        quote! {
+            impl #ident {
+                pub fn is_null(&self) -> bool {
+                    self.0.is_null()
+                }
```

Comment:
Types like `PSID` will get an `is_null` method instead of `is_invalid` when the underlying type is a pointer rather than an integer.